input/pagecontent/changes.md

```diff
@@ -66,35 +66,43 @@ This change log documents the significant updates and resolutions implemented fr
-- Removed Must Support from the following elements in AU Core Waist Circumference: 
-  - Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
-  - Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786)  
+- Made the following changes to AU Core Blood Pressure: 
+  - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
+  - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786)
+  - removed the fixed value constraint on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120)
+- Made the following changes to AU Core Body Height:
+  - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
+  - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786)
+  - removed the fixed value constraint on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120)
+- Made the following changes to AU Core Body Temperature: 
+  - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
+  - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786) 
+  - removed the fixed value constraint on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120)
+- Made the following changes to AU Core Body Weight: 
+  - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
+  - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786)
+  - removed the fixed value constraint on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120) 
+- Made the following changes to AU Core Heart Rate: 
+  - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
+  - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786) 
+  - removed the fixed value constraint on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120)
+- Made the following changes to AU Core Respiration Rate: 
+  - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
+  - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786)
+  - removed the fixed value constraint on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120)
+- Made the following changes to AU Core Waist Circumference: 
+  - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
+  - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786)  
+  - removed the fixed value constraint on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120)
```

Comment:
change "removed the fixed value constraint on Observation.status" to be "removed the fixed value constraint 'final' from Observation.status"

Reply:
Done.